modules/mobile/app/services/mobile/v0/disability_rating/proxy.rb

```diff
@@ -6,52 +6,29 @@ module Mobile
   module V0
     module DisabilityRating
       class Proxy
-        def initialize(user)
-          @user = user
+        def initialize(icn)
```

Comment:
What was the reasoning behind no longer passing the full user object through?

Reply:
the evss services the old one used required the whole user object while the new one just needs the icn.